bindings/python/iota_sdk/client/client.py

```diff
@@ -49,41 +52,42 @@ def __init__(
     ):
         """Initialize the IOTA Client.
 
-        Parameters
-        ----------
-        nodes : string or array of string
-            A single Node URL or an array of URLs.
-        primary_node : string
-            Node which will be tried first for all requests.
-        primary_pow_node : string
-            Node which will be tried first when using remote PoW, even before the primary_node.
-        permanode : string
+        **Arguments**
```

Comment:
Just `Args:`  and without "----"

Reply:
I can't use that here. It's limited to maximum of 10 items :cry: